samples/open-telemetry/jaeger/Core_8/Publisher/Publisher.csproj

```
@@ -8,8 +8,7 @@
 
 	<ItemGroup>
 		<PackageReference Include="NServiceBus" Version="8.*" />
-		<PackageReference Include="OpenTelemetry" Version="1.*" />
-		<PackageReference Include="OpenTelemetry.Exporter.Jaeger" Version="1.*" />
+    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.7.0" />
```

```suggestion
    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.*" />
```
Should this be a wildcard as well?